frontend/public/ui/ui-reducers.js

```diff
@@ -100,10 +102,14 @@ export default (state, action) => {
     case types.dismissOverviewDetails:
       return state.mergeIn(['overview'], {selectedUID: '', selectedDetailsTab: ''});
 
+    case types.updateOverviewMetrics:
+      return state.mergeIn(['overview', 'metrics'], action.metrics);
```

Comment:
Why `mergeIn` and not `setIn`? If there are stale metrics stored in this map, wouldn't this cause them not to be removed?